js/LogoView.js

```diff
@@ -8,11 +8,47 @@ define([
 
     initialize: function() {
       this.render();
+      this.listenTo(Adapt, "device:changed", this.onDeviceResize);
+    },
+
+    onDeviceResize: function() {
+      this.postRender();
+    },
+
+    postRender: function() {
+      var config = Adapt.course.get("_navigationLogo");
+      if (config && config._hideLogoForMobile) {
+        this.hideLogoForMobile();
+      }
+      if (config && config._graphic._mobileSrc) {
+        this.setLogoImageSrc();
+      }
+    },
+
+    hideLogoForMobile: function() {
+      (Adapt.device.screenSize === 'small')
+        // mobile logo display
+        ? $(".navigation-logo__image").addClass('u-display-none')
+        // desktop logo display
+        : $(".navigation-logo__image").removeClass('u-display-none');
+    },
+
+    setLogoImageSrc: function() {
+      var config = Adapt.course.get("_navigationLogo");
+      var mobileSrc = config._graphic._mobileSrc;
+      var src = config._graphic._src;
+      (Adapt.device.screenSize === 'small')
+        // mobile logo display
+        ? $(".navigation-logo__image").attr('src', mobileSrc)
+        // desktop logo display
+        : $(".navigation-logo__image").attr('src', src);
```

Comment:
Suggest assigning ternary to variable, e.g.
```js
var config = Adapt.course.get('_navigationLogo')._graphic;
var src = Adapt.device.screenSize === 'small' ?  config._mobileSrc : config._src;

$('.navigation-logo__image').attr('src', src);
```
You could also pass in `config` as a parameter from `postRender()`, negating the repeated look-up.